### ISSUE TITLE
Silent Login not working for ADAL Library in iOS

### ISSUE THREAD

**Issue by sin713** [AzureAD/azure-activedirectory-library-for-objc#1456]
After successful attempt of SSO Login and application kill and relaunch, silent login is not working.
Below is the attachment of code

What we are expecting: Application should  authenticate without asking username and password as SSO was successful.

<img width="1323" alt="Screenshot 2019-08-29 at 4 38 16 PM" src="https://user-images.githubusercontent.com/6995247/63935614-7b4ddf80-ca7b-11e9-824e-2b28af02b9c1.png">

**Comment by oldalton** [comment]
@sin713, I believe this issue has been resolved offline. Can you please confirm and close the issue?

**Comment by oldalton** [comment]
I believe this was app misconfiguration and it was resolved offline. @mdivkovic-speech, please ensure that you are configuring ADAL keychain access groups correctly and that you are not clearing keychain yourself.

**Comment by mdivkovic-speech** [comment]
@oldalton, thank you very much for the reply, I have been able to resolve the issue in the meantime. I realized that some of the fields (e.g. authority, resource) were not getting persisted after restarting the app, and then the SDK was called with an empty string instead. I did not see any error messages in the logs, and my completion block was never called. After making sure to persist all the inputs for the auth context and silent login call, it succeeded.
